software/bsg_manycore_lib/bsg_manycore_asm.h

```diff
@@ -99,4 +99,71 @@
     li t0, val;             \
     op reg,t0,dest;
 
+// start code
+#define bsg_asm_start  \
```

Comment:
can we rename it to something more descriptive? e.g. bsg_asm_init_regfile

Reply:
Done